src/backend/distributed/utils/citus_clauses.c

```diff
@@ -213,13 +235,13 @@ citus_evaluate_expr(Expr *expr, Oid result_type, int32 result_typmod,
 
 		if (IsA(expr, Param))
 		{
-			if (!masterEvaluationContext->evaluateParams)
+			if (masterEvaluationContext->evaluationMode == EvaluateNone)
```

Comment:
Maybe just omit EvaluateNone since we never use it?

Reply:
We use it in `citus_evaluate_expr()`. I find it a little simpler to have `EvaluateNone `.

The two existing APIs evaluate `PARAM`s at the moment. We could potentially get rid of `EvaluateNone` and change  the ` bail out` parts  in `citus_evaluate_expr`  to something like

```
if (ShouldEvaluateExpressionType() && masterEvaluationContext->evaluationMode != EvaluateFunctionsAndParams)
{
 /* bail out*/
}
else  if (Param)
{
    /* continue with evaluation, we currently always  evaluate PARAMs  */   
}
```
As I said, I've a slight preference to keep `EvaluateNone `, but if that's  confusing I can change it